### ISSUE TITLE
profiler seems not print all op calls

### ISSUE THREAD

**Issue by COthello** [pytorch/pytorch#20629]
## 🐛 Bug

Hello everyone, I use pytorch profiler to get my program performance, but seems profiler not print all op function calls. 
My program has 53 conv2d, but profiler only print 6 conv2d calls, can anyone help on that? Thanks!
![image](https://user-images.githubusercontent.com/9839536/57908296-4645c480-78b2-11e9-8a57-24c5e771c60b.png)

## To Reproduce

                 with torch.autograd.profiler.profile(use_cuda=True) as prof:
                    outputs = model(inputs)
                    _, preds = torch.max(outputs.data, 1)
                    loss = criterion(outputs, labels)
                 print(prof)
## Environment

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
(or fill out the checklist below manually).

You can get the script and run it with:
```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```

 - PyTorch Version (e.g., 1.0):1.1.0a0+e8fb5f3
 - OS (e.g., Linux):Linux
 - How you installed PyTorch (`conda`, `pip`, source): source
 - Build command you used (if compiling from source): python setup.py install
 - Python version: 3.5
 - CUDA/cuDNN version: 9.2
 - GPU models and configuration:
 - Any other relevant information:

## Additional context

cc @ezyang @SsnL @albanD @zou3519 @gqchen

**Comment by aaronenyeshi** [comment]
There was a significant refactor of the profiler. This should be fixed.
